AWSCognitoIdentityProvider/AWSCognitoIdentityUserPool.h

```diff
@@ -56,10 +56,12 @@ NS_ASSUME_NONNULL_BEGIN
                                    userPoolConfiguration:(AWSCognitoIdentityUserPoolConfiguration *)userPoolConfiguration
                                                   forKey:(NSString *)key;
 
-+ (instancetype)CognitoIdentityUserPoolForKey:(NSString *)key;
++ (nullable instancetype)CognitoIdentityUserPoolForKey:(NSString *)key;
```

Comment:
Breaking change, please revert.

Reply:
Changed. Had to change the way this is used.

```
let client = AWSCognitoIdentityUserPool.init(forKey: clientKey)
if (client == nil) {
...
}
```